configure.ac

```diff
@@ -264,26 +280,66 @@ RKT_ITERATE_FLAVORS([${RKT_STAGE1_FLAVORS}],[flavor],
                      RKT_SPECIFIED_FLAVORS="${RKT_SPECIFIED_FLAVORS},${flavor}"
                      AS_CASE([${flavor}],
                              [src],
-                                     [RKT_REQ_PROG([INTLTOOLIZE],[intltoolize],[intltoolize])
-                                      RKT_REQ_PROG([LIBTOOLIZE],[libtoolize],[libtoolize])],
+                                     [AC_MSG_NOTICE([will build systemd from source, make sure that all its build requirements are fulfilled])],
                              [coreos],
-                                     [RKT_REQ_PROG([WGET],[wget],[wget])
-                                      RKT_REQ_PROG([MKTEMP],[mktemp],[mktemp])
-                                      RKT_REQ_PROG([MD5SUM],[md5sum],[md5sum])
-                                      RKT_REQ_PROG([CPIO],[cpio],[cpio])
-                                      RKT_REQ_PROG([GZIP],[gzip],[gzip])
-                                      RKT_REQ_PROG([UNSQUASHFS],[unsquashfs],[unsquashfs])
-                                      RKT_REQ_PROG([SORT],[sort],[sort])],
+                                     [RKT_COMMON_COREOS_PROGS],
                              [kvm],
                                      [AC_MSG_WARN([* kvm is an experimental stage1 implementation, some features are missing])
+                                      AC_MSG_NOTICE([will build linux kernel and lkvm from source, make sure that all their build requirements are fulfilled])
+                                      RKT_COMMON_COREOS_PROGS
                                       RKT_REQ_PROG([PATCH],[patch],[patch])
-                                      RKT_REQ_PROG([BC],[bc],[bc])],
```

Comment:
Is `bc` not needed anymore?

I see it is still listed as a dependency elsewhere:
- scripts/install-deps-debian-sid.sh
- scripts/install-deps-fedora-22.sh